src/Compilers/VisualBasic/Test/Semantic/Semantics/AnonymousTypesTests.vb

```diff
@@ -586,13 +574,13 @@ End Module]]>.Value
             Dim expectedOperationTree = <![CDATA[
 IAnonymousObjectCreationExpression (OperationKind.AnonymousObjectCreationExpression, Type: <anonymous type: a As System.Object>) (Syntax: 'New With {.a = Nothing}')
   Initializers(1):
-      ISimpleAssignmentExpression (OperationKind.SimpleAssignmentExpression, Type: System.Object, Constant: null) (Syntax: '.a = Nothing')
+      ISimpleAssignmentExpression (OperationKind.SimpleAssignmentExpression, Type: System.Object, Constant: null, IsImplicit) (Syntax: '.a = Nothing')
```

Comment:
>IsImplicit [](start = 114, length = 10)

It is not clear why this node became Implicit, feels unexpected.

Reply:
Ok, so the reason is this line:
http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Operations/VisualBasicOperationFactory_Methods.vb,200

value in this example is the BoundConversion from Nothing to Object, which is implicit. Previously, this conversion was not implicit, so the node wasn't marked as implicit. I believe this is a bug in GetObjectCreationInitializers, and not related to this PR. Filed https://github.com/dotnet/roslyn/issues/22588.